electron/ironfish/AccountManager.ts

```diff
@@ -306,6 +306,7 @@ class AccountManager
         }
       : createParams
     const newAccount = await this.node.wallet.importAccount(create)
+    this.node.wallet.scanTransactions()
```

Comment:
I wonder whether we should push this logic from the `importAccount` into the `wallet.importAccount` method 🤔

Reply:
Good question, I think it is nice to keep it modular though. I can imagine a situation where you want to add an account but not start scanning. Like when this app is booting up, lets say you want to add multiple accounts in a row, it would be nice to add them all then start the scan.